### PR TITLE
Python 3: Renames unicode to str

### PR DESCRIPTION
**Pull request by hsmj1412**
Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

File: selftests/unit/test_libvirt_xml.py [hunk 1]
```diff
@@ -9,6 +9,12 @@
 from avocado.utils import process
 
 
+try:
+    unicode
+except ImportError:
```
Comment (chunfuwen) [inline selftests/unit/test_libvirt_xml.py]: ImportError to NameError

File: virttest/staging/backports/simplejson/decoder.py [hunk 2]
```diff
@@ -6,6 +6,11 @@
 
 from scanner import make_scanner
 
+
+try:
+    unicode
+except ImportError:
```
Comment (chunfuwen) [inline virttest/staging/backports/simplejson/decoder.py]: ImportError to NameError

File: virttest/utils_net.py [hunk 3]
```diff
@@ -26,6 +26,13 @@
 from . import arch
 from .versionable_class import factory
 
+
+try:
+    unicode
+except ImportError:
```
Comment (chunfuwen) [inline virttest/utils_net.py]: ImportError to NameError

**Comment by hsmj1412** [comment]
@chunfuwen Done

**Pull request merged**